src/foam/nanos/auth/CreatedByAwareMixin.js

```diff
@@ -0,0 +1,33 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.nanos.auth',
+  name: 'CreatedByAware',
+
+  implements: [
+    'foam.nanos.auth.CreatedByAware'
+  ],
+
+  properties: [
```

Comment:
Can you also include the TableCellFormatters to show the user legalname?

Reply:
I think we could refine Reference itself to display the toSummary() of the referenced object, then we wouldn't have to do it here (or anywhere else).

Reply:
Actually, I don't think that was a good idea. In some cases, like a User's group, the group id is sufficient so no lookup or advanced formatter is required. We could however, make a generic ReferenceToSummaryFormatter which could be dropped in wherever we need it.

Reply:
The formatter works, but group is the outlier. The majority of references are long or guid ids.

Reply:
I'm working on a generic Reference TableCellRenderer, which can be part of a future PR.